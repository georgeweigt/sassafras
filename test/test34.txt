data
input trt $ x y
datalines
M-4	20	12.8
M-4	17	11
M-4	20	10.9
M-4	15	9.1
M-4	20	9.6
M-4	15	9.3
360	20	12.2
360	20	10
360	16	9.8
360	20	9.8
360	19	9.8
360	20	12.1
K-24	20	14.1
K-24	20	13.1
K-24	20	12.8
K-24	20	11.8
K-24	20	10.8
K-24	13	7.8
M-15	13	8.6
M-15	18	10.2
M-15	17	8.7
M-15	14	7.3
M-15	15	9.3
M-15	11	8.2
;

proc means mean clm
var y
class trt

proc reg
model y = trt x / noint