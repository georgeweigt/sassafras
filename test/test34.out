          TRT      Variable       Mean     95% CLM MIN     95% CLM MAX
          M-4      Y            10.450           8.974          11.926
          360      Y            10.617           9.367          11.866
          K-24     Y            11.733           9.389          14.078
          M-15     Y             8.717           7.685           9.748

                              Analysis of Variance

               DF     Sum of Squares     Mean Square     F Value     Pr > F
     Model      4           49.68324        12.42081        9.43     0.0002
     Error     19           25.03634         1.31770                       
     Total     23           74.71958                                       

              Root MSE            1.14791     R-Square     0.6649
              Dependent Mean     10.37917     Adj R-Sq     0.5944
              Coeff Var          11.05977                        

                              Parameter Estimates

                        Estimate     Std Err     t Value     Pr > |t|
           TRT M-4       2.79201     1.94321        1.44       0.1670
           TRT 360       2.38612     2.08032        1.15       0.2656
           TRT K-24      3.64592     2.04599        1.78       0.0907
           TRT M-15      2.41851     1.62023        1.49       0.1519
           X             0.42942     0.10575        4.06       0.0007

